ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -281,6 +281,16 @@ interface {{ ethernet_interface.name }}
    ip address {{ ip_address_secondary }} secondary
 {%                 endfor %}
 {%             endif %}
+{%             for ip_helper in ethernet_interface.ip_helpers | arista.avd.convert_dicts('ip_helper') | arista.avd.natural_sort('ip_helper') %}
```

Comment:
Since you were adding a new feature. Please use list of dictionaries instead of the dictionaries. This will make sure we don't need to add ```arista.avd.convert_dicts('ip_helper')```, which is a dictionary to list converter.

Reply:
Not sure I agree with that statement as this is not really a "new feature" - for now the PR is copying over the current model under VLAN to Ethernet interfaces - I would argue it is best until we actually move to 4.0.0 model to keep a unified model under VLAN and Ethernet so that it is not confusing for the user to have one syntax documented under VLAN and another documented under Ethernet in 3.x.